pkg/virt/vm.go

```diff
@@ -24,6 +24,7 @@ import (
 type VM struct {
 	Instance   libvirt.Domain
 	Descriptor libvirtxml.Domain
+	Logger     Logger
```

Comment:
Is there an specific reason why you changed the data type from a pointer type to a non-pointer type?

Reply:
Because it’s not needed, one part of the interface will point to whatever data we pass to it. See [this question on SO](https://stackoverflow.com/a/27178682) for a quick summary

Reply:
Because it’s not needed, one part of the interface will point to whatever data we pass to it. See [this question on SO](https://stackoverflow.com/a/27178682) for a quick summary

Reply:
Woah, golang still suprises me at some points! Thank your for clarification.